lib/presentation/widget/login/login_widget.dart

```diff
@@ -192,13 +177,19 @@ class _LoginWidgetState extends State<LoginWidget> {
                   ),
                 ),
               ),
+              ToolbarBuilder()
```

Comment:
I think u can remove the back button for now. Because it currently going no where (since we don't have introduction screen yet)